src/rust/ide/src/double_representation/definition.rs

```diff
@@ -148,6 +119,14 @@ impl DefinitionName {
         DefinitionName {name, extended_target:default()}
     }
 
+    /// Creates a new name consisting of a single identifier, without any extension target.
```

Comment:
What is extension target? I mean, this might be a common knowledge here, but it would be super cool if docs would refer to it. I try to always point in my docs to the right place by at least mentioning a type like "without an `ExtensionTarget`" - if something like `ExtensionTarget` is in scope and has the right documentation. If its not, then I believe docs should explain that or guide me to the place where I can learn more about it.